src/UnitTests/IMappingExpression/NonGenericProjectEnumTest.cs

```diff
@@ -73,7 +73,7 @@ public void ProjectingEnumToString()
         {
             var customers = new[] { new Customer() { FirstName = "Bill", LastName = "White", CustomerType = CustomerType.Vip } }.AsQueryable();
 
-            var projected = _mapper.Map<CustomerDto[]>(customers);
+            var projected = customers.ProjectTo<CustomerDto>(_mapper.ConfigurationProvider);
```

Comment:
@jbogard I assume this is what it should of been.  I don't know of mapping IQueryable should enforce projection instead of calling one of the enumerable mappers.

What was funny though was when I refactored Enum Mapper it failed.  But only when I had IObjectMapExpression activated.  It passed fine with just IObjectMapper though.